doc/Caliptra.md

```diff
@@ -979,35 +979,35 @@ For SoCs that intend to achieve FIPS 140-3 CMVP certification with Caliptra:
 #### Fuse map
 The following table describes Caliptra's fuse map:
 
-*Table 13: Caliptra Fuse Map*
+*Table 12: Caliptra Fuse Map*
 
 | **Name**                        | **Size (bits)** | **ACL**         | **Fuse programming time**                       | **Description** |
 | ------------------------------- | --------------- | --------------- | ----------------------------------------------- | --------------- |
 | UDS Seed (obfuscated)           | 384             | ROM             | SoC manufacturing                               | DICE Unique Device Secret Seed. This seed is unique per device. The seed is scrambled using an obfuscation function. |
-| Field Entropy (obfuscated)      | 256             | ROM             | <p>Device owner</p><p>In-field programmable</p> | Field-programmable by the owner, used to hedge against UDS disclosure in the supply chain. |
+| Field Entropy (obfuscated)      | 256             | ROM             | Device owner in-field programmable | Field-programmable by the owner, used to hedge against UDS disclosure in the supply chain. |
 | KEY MANIFEST PK HASH            | 384             | ROM FMC RUNTIME | SoC manufacturing                               | SHA-384 hash of hashes of the Key Manifest Signing the ECC P-384 Public Keys. |
 | KEY MANIFEST PK HASH MASK       | 4               | ROM FMC RUNTIME | In-field programmable                           | One-hot encoded list of revoked Key Manifest PK Hash. |
 | OWNER PK HASH                   | 384             | ROM FMC RUNTIME | In-field programmable                           | SHA-384 hash of hashes of the Owner Key ECC P-384 Public Keys. |
-| OWNER KEY MANIFEST PK HASH MASK | 4               | ROM FMC RUNTIME | In-field programmable                           | <p>One-hot encoded list of revoked Owner Key Manifest PK Hash.</p><p>**Reset:**</p> |
+| OWNER KEY MANIFEST PK HASH MASK | 4               | ROM FMC RUNTIME | In-field programmable                           | One-hot encoded list of revoked Owner Key Manifest PK Hash. **Reset** |
```

Comment:
The **Reset** seems like a bug. Perhaps someone wanted to add a **Reset** column. Recommend just deleting it in this PR.

Reply:
done